src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Unix.cs

```diff
@@ -61,6 +61,7 @@ public static partial class PlatformDetection
         public static bool IsNetfx472OrNewer => false;
 
         public static bool IsDrawingSupported { get; } = GetGdiplusIsAvailable();
+        public static bool IsSoundPlaySupported { get; } = false;
```

Comment:
It is not supported in Unix either?

Reply:
Looking at `Configuration.props` file for test project is built for Windows only, I decided to set this false for Unix as well.

Reply:
Sounds reasonable then!